zuul.d/jobs.yaml

```diff
@@ -62,9 +62,9 @@
     secrets:
       - secret: zuul_eco_project_config_vault_new
         name: vault_data
-    nodeset: ubuntu-focal
+    nodeset: ubuntu-jammy
     vars:
-      go_version: "1.18"
+      go_version: 1.18.1
```

Comment:
please leave the quotes. Fun starts once you put something like 1.10 (which will magically converts into 1.1)